### DISCUSSION TITLE
Force dark-mode on a specific page

### DISCUSSION THREAD

**Discussion by botkero** [squidfunk/mkdocs-material#5788] [Q&A] [answered]
Would it be possible to force the dark-mode on a specific page?

For example:

```md
---
scheme: slate
---

# Header1

Lorem ipsum...
```

**Comment by squidfunk** [comment, marked as answer]
No, that's not supported and not planned, but you might achieve it with some additional JavaScript to automatically change the mode when the page loads. It's a rather unusual use case, which is why we're not considering it. We hope for your understanding.
